### ISSUE TITLE
Clicking an error for a Likert in ErrorReport does not navigate to the error

### ISSUE THREAD

**Issue by mikaelrss** [Altinn/app-frontend-react#1779]
### Description of the bug

When clicking an error in `ErrorReport` the component that produced the error should be focused. This does not happen for `Likert`

### Steps To Reproduce

1. Go to a page with a Likert.
2. Make the Likert required.
3. Navigate to the next page and see that there are error messages in the ErrorReport at the bottom. 
4. Click the error and see that the Likert is not focused. 

### Additional Information

This issue har a PR for a fix in v3: https://github.com/Altinn/app-frontend-react/pull/1778
